docs/DeveloperGuide.md

```diff
@@ -193,11 +193,11 @@ It implements the following operations:
 * `CountryNotesManager#addCountryNote(CountryNote countryNote)`  — Adds the given `countryNote` to the internal `ObservableList<CountryNote>`.
 * `CountryNotesManager#deleteCountryNote(CountryNote countryNote)`  — Deletes the given `countryNote` from the internal `ObservableList<CountryNote>`.
 
-The following class diagram illustrates how the relevant classes in the `seedu.address.model` package are related to `CountryNote`.
+The following class diagram illustrates how the relevant classes in the `Model` component are related to `CountryNote`.
 
 ![Country Class Diagram](images/CountryClass.png)
 
-Given below is a sequence diagram that shows how the `country note add` command works.
+Given below is a sequence diagram that shows how the `country note add` command works within the `Logic` component.
```

Comment:
I think can leave out the "within the `Logic` component", since the diagram also involves the model component. Also, not sure why the model part was cut to become less detailed?

Reply:
Can ref to the current DG in our website i think the diagram is quite stretched and the model component doesn't really capture any complex logic as its just nested calls.

Reply:
Ok sure